### ISSUE TITLE
Changelog deletes unknown sections

### ISSUE THREAD

**Issue by scotthavens** [USDA-ARS-NWRC/inicheck#47]
Giving an `awsm` config file to `inicheck` with just the `smrf` module will delete all non `smrf` sections. Since it should only really change the `smrf` sections, either need to have a warning that all the other sections will be deleted or an option to include unknown sections.

**Comment by micahjohnson150** [comment]
Yeah, I kind of remember something along those lines. What version of inicheck are you using?

**Comment by scotthavens** [comment]
It was the latest version

**Comment by micahjohnson150** [comment]
And did you use the command line or what this a result of how it is being used in AWSM/SMRF?

e.g. inicheck -f config.ini -m smrf awsm --change

**Comment by scotthavens** [comment]
@jomey may be able to provide the offending config file

**Comment by micahjohnson150** [comment]
Ok. I think I see where this is  happening. I need to check if a section is reference by both the master and changelog. If not do nothing with it.

**Comment by jomey** [comment]
If my memory is correct, then we used this command:
```
inicheck -f wy2019_gunnison_awsm.ini -m smrf --change -w
```

attached the file as well.
[wy2019_gunnison_awsm.ini.zip](https://github.com/USDA-ARS-NWRC/inicheck/files/4338755/wy2019_gunnison_awsm.ini.zip)
